Dockerfile

```diff
@@ -18,9 +18,8 @@ RUN go mod download
 COPY . /adam/src
 
 ARG GOOS=linux
-ARG GOARCH=amd64
```

Comment:
Also, seems, that we need to remove this line now. We do not use build-args, but we have action to build with defined platform (linux/arm64).

Reply:
Does having this line _break_ anything?

Reply:
Yes, it break building for arm64 inside action (we compile amd64 binary in all cases).

Reply:
Understood.

Considering how quick and painless CI for adam is, we should not intermix these. One PR for bumping alpine, one for adding/removing build ARGs. These are unrelated changes (just like the ldflags is an unrelated change).

Please remove it from here, keep this PR really simple, we can merge it in, and then we can open a separate small PR for that.

Reply:
Done and created new one #70